### ISSUE TITLE
Problem in the hebrew language

### ISSUE THREAD

**Issue by OVOOLIFE** [ruffle-rs/ruffle#16653]
### Describe the bug

The letters in the hebrew language are flipped. instead of "שלום" it writes as "םולש".
![צילום מסך 2024-06-09 172625](https://github.com/ruffle-rs/ruffle/assets/117748445/ad1a8781-7ca2-43d3-b7c7-8ad13a8203ad)

### Expected behavior

it should be קובץ, פקדים, סימניות ext.

### Content Location

not related to a spesific flash content

### Affected platform

Desktop app

### Operating system

Windows 11

**Comment by kjarosh** [comment]
Duplicate of https://github.com/ruffle-rs/ruffle/issues/3921

**Comment by kjarosh** [comment]
Whoops, just realized that https://github.com/ruffle-rs/ruffle/issues/3921 refers to RTL text inside SWFs, not in the GUI. I meant to mention https://github.com/ruffle-rs/ruffle/issues/11538, but as that issue refers to Arabic specifically, I'm reopening this one.

TL;DR We need egui support for RTL text https://github.com/emilk/egui/issues/1016
